.github/workflows/llvm.yml

```diff
@@ -48,11 +48,18 @@ jobs:
   windows-x86:
     runs-on: windows-2019
     steps:
-      - uses: supergrecko/javacpp-presets/.github/actions/deploy-windows@cmake-3-16
+      - uses: bytedeco/javacpp-presets/.github/actions/deploy-windows@actions
   windows-x86_64:
     runs-on: windows-2019
+    env:
+      BUILD_DIR: C:\build
     steps:
-      - uses: supergrecko/javacpp-presets/.github/actions/deploy-windows@cmake-3-16
+      - uses: al-cheb/configure-pagefile-action@v1.2
+        with:
+          minimum-size: 8GB
+          maximum-size: 16GB
+          disk-root: "C:"
+      - uses: bytedeco/javacpp-presets/.github/actions/deploy-windows@actions
```

Comment:
Not the pagefile directory, the build directory of CMake.

Reply:
Right, what would that look like? I just grabbed this from the Pytorch preset file which had CI setup like that.

Reply:
Usually with CMake we just need to `cd ...` to a different directory before calling `cmake ...`, and I don't think LLVM tries to do anything fancy here like we need to do for PyTorch.

Reply:
So you suggest we just cd into the C:\ disk and build everything in there?